scripts/jenkins/Jenkinsfile

```diff
@@ -773,24 +817,50 @@ def xunitUpload() {
     ])
 }
 
+/* Create a new Docker Image Description
+ *
+ * @param name Name of the image, will be extended with registry, a common
+ *             prefix and a tag
+ * @param tagFun Closure describing how the tag should be formatted
+ * @param context Context for the docker build
+ * @param file Path to Dockerfile
```

Comment:
relative to?

Reply:
project root (so a checkout of the repo)

Reply:
Everything else does not make sense hence i did not point it out (after all you are all the time moving in a checkout of the repository) + one can see it from the existing description calls.

Do you want to state it explicitly?

Reply:
You can also say it at one place in BUILDSERVER.md or beginning of the Jenkinsfile.

Reply:
actually i was wrong. it is dependent on the location the script is currently in so such an statement would be wrong.